# STEP3: Perform some simple data analysis

Start by connecting to the database by running the cells below. If you are coming back to this exercise, then uncomment and run the first cell to recreate the database. If you recently completed steps 1 and 2, then skip to the second cell.

In [2]:
#!PGPASSWORD=student createdb -h 127.0.0.1 -U student pagila
#!PGPASSWORD=student psql -q -h 127.0.0.1 -U student -d pagila -f Data/pagila-schema.sql
#!PGPASSWORD=student psql -q -h 127.0.0.1 -U student -d pagila -f Data/pagila-data.sql

In [1]:
%load_ext sql

DB_ENDPOINT = "127.0.0.1"
DB = 'pagila'
DB_USER = 'student'
DB_PASSWORD = 'student'
DB_PORT = '5432'

# postgresql://username:password@host:port/database
conn_string = "postgresql://{}:{}@{}:{}/{}" \
                        .format(DB_USER, DB_PASSWORD, DB_ENDPOINT, DB_PORT, DB)

print(conn_string)
%sql $conn_string

postgresql://student:student@127.0.0.1:5432/pagila


'Connected: student@pagila'

### 3NF - Entity Relationship Diagram

<img src="./pagila-3nf.png" width="50%"/>

## 3.1 Insight 1:   Top Grossing Movies 
- Payments amounts are in table `payment`
- Movies are in table `film`
- They are not directly linked, `payment` refers to a `rental`, `rental` refers to an `inventory` item and `inventory` item refers to a `film`
- `payment` &rarr; `rental` &rarr; `inventory` &rarr; `film`

### 3.1.1 Films

In [2]:
%%sql
select film_id, title, release_year, rental_rate, rating  
from film
limit 5;

 * postgresql://student:***@127.0.0.1:5432/pagila
5 rows affected.


[(1, 'ACADEMY DINOSAUR', 2006, Decimal('0.99'), 'PG'),
 (2, 'ACE GOLDFINGER', 2006, Decimal('4.99'), 'G'),
 (3, 'ADAPTATION HOLES', 2006, Decimal('2.99'), 'NC-17'),
 (4, 'AFFAIR PREJUDICE', 2006, Decimal('2.99'), 'G'),
 (5, 'AFRICAN EGG', 2006, Decimal('2.99'), 'G')]

### 3.1.2 Payments

In [3]:
%%sql
select * 
from payment 
limit 5;

 * postgresql://student:***@127.0.0.1:5432/pagila
5 rows affected.


[(16050, 269, 2, 7, Decimal('1.99'), datetime.datetime(2017, 1, 24, 21, 40, 19, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None))),
 (16051, 269, 1, 98, Decimal('0.99'), datetime.datetime(2017, 1, 25, 15, 16, 50, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None))),
 (16052, 269, 2, 678, Decimal('6.99'), datetime.datetime(2017, 1, 28, 21, 44, 14, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None))),
 (16053, 269, 2, 703, Decimal('0.99'), datetime.datetime(2017, 1, 29, 0, 58, 2, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None))),
 (16054, 269, 1, 750, Decimal('4.99'), datetime.datetime(2017, 1, 29, 8, 10, 6, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)))]

### 3.1.3 Inventory

In [4]:
%%sql
select * 
from inventory 
limit 5;

 * postgresql://student:***@127.0.0.1:5432/pagila
5 rows affected.


[(1, 1, 1, datetime.datetime(2017, 2, 15, 10, 9, 17, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None))),
 (2, 1, 1, datetime.datetime(2017, 2, 15, 10, 9, 17, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None))),
 (3, 1, 1, datetime.datetime(2017, 2, 15, 10, 9, 17, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None))),
 (4, 1, 1, datetime.datetime(2017, 2, 15, 10, 9, 17, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None))),
 (5, 1, 2, datetime.datetime(2017, 2, 15, 10, 9, 17, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)))]

### 3.1.4 Get the movie of every payment

In [5]:
%%sql
SELECT f.title, p.amount, p.payment_date, p.customer_id                                            
FROM payment p
JOIN rental r    ON ( p.rental_id = r.rental_id )
JOIN inventory i ON ( r.inventory_id = i.inventory_id )
JOIN film f ON ( i.film_id = f.film_id)
limit 5;

 * postgresql://student:***@127.0.0.1:5432/pagila
5 rows affected.


[('SWARM GOLD', Decimal('1.99'), datetime.datetime(2017, 1, 24, 21, 40, 19, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 269),
 ('PACKER MADIGAN', Decimal('0.99'), datetime.datetime(2017, 1, 25, 15, 16, 50, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 269),
 ('SOMETHING DUCK', Decimal('6.99'), datetime.datetime(2017, 1, 28, 21, 44, 14, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 269),
 ('DRACULA CRYSTAL', Decimal('0.99'), datetime.datetime(2017, 1, 29, 0, 58, 2, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 269),
 ('CLOSER BANG', Decimal('4.99'), datetime.datetime(2017, 1, 29, 8, 10, 6, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 269)]

### 3.1.5 sum movie rental revenue
TODO: Write a query that displays the amount of revenue from each title. Limit the results to the top 10 grossing titles. Your results should match the table below.

In [6]:
%%sql
SELECT f.title, sum(p.amount) as Revenue
FROM (SELECT DISTINCT payment_id, rental_id, amount FROM payment) p
JOIN rental r    ON ( p.rental_id = r.rental_id )
JOIN inventory i ON ( r.inventory_id = i.inventory_id )
JOIN film f ON ( i.film_id = f.film_id)
GROUP BY f.title
ORDER BY sum(p.amount) DESC
limit 10;

 * postgresql://student:***@127.0.0.1:5432/pagila
10 rows affected.


[('TELEGRAPH VOYAGE', Decimal('231.73')),
 ('WIFE TURN', Decimal('223.69')),
 ('ZORRO ARK', Decimal('214.69')),
 ('GOODFELLAS SALUTE', Decimal('209.69')),
 ('SATURDAY LAMBS', Decimal('204.72')),
 ('TITANS JERK', Decimal('201.71')),
 ('TORQUE BOUND', Decimal('198.72')),
 ('HARRY IDAHO', Decimal('195.70')),
 ('INNOCENT USUAL', Decimal('191.74')),
 ('HUSTLER PARTY', Decimal('190.78'))]

<div class="p-Widget jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/html"><table>
    <tbody><tr>
        <th>title</th>
        <th>revenue</th>
    </tr>
    <tr>
        <td>TELEGRAPH VOYAGE</td>
        <td>231.73</td>
    </tr>
    <tr>
        <td>WIFE TURN</td>
        <td>223.69</td>
    </tr>
    <tr>
        <td>ZORRO ARK</td>
        <td>214.69</td>
    </tr>
    <tr>
        <td>GOODFELLAS SALUTE</td>
        <td>209.69</td>
    </tr>
    <tr>
        <td>SATURDAY LAMBS</td>
        <td>204.72</td>
    </tr>
    <tr>
        <td>TITANS JERK</td>
        <td>201.71</td>
    </tr>
    <tr>
        <td>TORQUE BOUND</td>
        <td>198.72</td>
    </tr>
    <tr>
        <td>HARRY IDAHO</td>
        <td>195.70</td>
    </tr>
    <tr>
        <td>INNOCENT USUAL</td>
        <td>191.74</td>
    </tr>
    <tr>
        <td>HUSTLER PARTY</td>
        <td>190.78</td>
    </tr>
</tbody></table></div>

## 3.2 Insight 2:   Top grossing cities 
- Payments amounts are in table `payment`
- Cities are in table `cities`
- `payment` &rarr; `customer` &rarr; `address` &rarr; `city`

### 3.2.1 Get the city of each payment

In [7]:
%%sql
SELECT p.customer_id, p.rental_id, p.amount, ci.city                            
FROM payment p
JOIN customer c  ON ( p.customer_id = c.customer_id )
JOIN address a ON ( c.address_id = a.address_id )
JOIN city ci ON ( a.city_id = ci.city_id )
order by p.payment_date
limit 10;

 * postgresql://student:***@127.0.0.1:5432/pagila
10 rows affected.


[(130, 1, Decimal('2.99'), 'guas Lindas de Gois'),
 (459, 2, Decimal('2.99'), 'Qomsheh'),
 (408, 3, Decimal('3.99'), 'Jaffna'),
 (333, 4, Decimal('4.99'), 'Baku'),
 (222, 5, Decimal('6.99'), 'Jaroslavl'),
 (549, 6, Decimal('0.99'), 'Santiago de Compostela'),
 (269, 7, Decimal('1.99'), 'Salinas'),
 (239, 8, Decimal('4.99'), 'Ciomas'),
 (126, 9, Decimal('4.99'), 'Po'),
 (399, 10, Decimal('5.99'), 'Okara')]

### 3.2.2 Top grossing cities
TODO: Write a query that returns the total amount of revenue by city as measured by the `amount` variable in the `payment` table. Limit the results to the top 10 cities. Your result should match the table below.

In [11]:
%%sql
SELECT ci.city, sum(p.amount) as revenue
FROM payment p
JOIN customer c  ON ( p.customer_id = c.customer_id )
JOIN address a ON ( c.address_id = a.address_id )
JOIN city ci ON ( a.city_id = ci.city_id )
group by ci.city
order by sum(p.amount) desc

 * postgresql://student:***@127.0.0.1:5432/pagila
597 rows affected.


[('Cape Coral', Decimal('221.55')),
 ('Saint-Denis', Decimal('216.54')),
 ('Aurora', Decimal('198.50')),
 ('Molodetno', Decimal('195.58')),
 ('Santa Brbara dOeste', Decimal('194.61')),
 ('Apeldoorn', Decimal('194.61')),
 ('Qomsheh', Decimal('186.62')),
 ('London', Decimal('180.52')),
 ('Ourense (Orense)', Decimal('177.60')),
 ('Bijapur', Decimal('175.61')),
 ('Tanza', Decimal('175.58')),
 ('Memphis', Decimal('174.66')),
 ('Skikda', Decimal('173.63')),
 ('Halisahar', Decimal('170.67')),
 ('Omdurman', Decimal('169.65')),
 ('Santa Rosa', Decimal('167.67')),
 ('Richmond Hill', Decimal('167.62')),
 ('Valparai', Decimal('166.65')),
 ('Kolpino', Decimal('162.62')),
 ('Bhilwara', Decimal('161.68')),
 ('Tanauan', Decimal('161.65')),
 ('Hodeida', Decimal('160.68')),
 ('Sumy', Decimal('159.72')),
 ('Zhoushan', Decimal('159.68')),
 ('Usolje-Sibirskoje', Decimal('159.67')),
 ('Lengshuijiang', Decimal('159.64')),
 ('Donostia-San Sebastin', Decimal('158.69')),
 ('Probolinggo', Decimal('158.66')),
 ('Salamanca', Decimal('157.65')),
 ('Najafabad', Decimal('157.65')),
 ('Maikop', Decimal('155.68')),
 ('Tangail', Decimal('155.65')),
 ('Changhwa', Decimal('155.59')),
 ('Changzhou', Decimal('154.60')),
 ('Shubra al-Khayma', Decimal('153.66')),
 ('Cam Ranh', Decimal('152.68')),
 ('Nakhon Sawan', Decimal('152.67')),
 ('Citt del Vaticano', Decimal('152.66')),
 ('Karnal', Decimal('152.65')),
 ('Pontianak', Decimal('152.65')),
 ('Balikesir', Decimal('152.63')),
 ('Kragujevac', Decimal('151.67')),
 ('Uruapan', Decimal('151.67')),
 ('Fuyu', Decimal('151.66')),
 ('Mannheim', Decimal('151.65')),
 ('Bergamo', Decimal('151.64')),
 ('Beira', Decimal('151.63')),
 ('Arecibo', Decimal('150.66')),
 ('Cabuyao', Decimal('149.65')),
 ('Yangor', Decimal('148.69')),
 ('Songkhla', Decimal('148.67')),
 ('Brescia', Decimal('147.69')),
 ('Usak', Decimal('147.65')),
 ('Ilorin', Decimal('146.69')),
 ('Akron', Decimal('146.69')),
 ('Hsichuh', Decimal('146.64')),
 ('Sincelejo', Decimal('145.70')),
 ('Belm', Decimal('145.68')),
 ('Batman', Decimal('145.67')),
 ('Vancouver', Decimal('144.70')),
 ('Ogbomosho', Decimal('144.68')),
 ('Naju', Decimal('144.67')),
 ('Malm', Decimal('144.66')),
 ('Nantou', Decimal('144.62')),
 ('Onomichi', Decimal('143.71')),
 ('Nyeri', Decimal('143.71')),
 ('Dos Quebradas', Decimal('143.68')),
 ('Kowloon and New Kowloon', Decimal('142.70')),
 ('Daugavpils', Decimal('142.70')),
 ('Moscow', Decimal('142.66')),
 ('Ciomas', Decimal('142.66')),
 ('Vila Velha', Decimal('141.71')),
 ('Nam Dinh', Decimal('141.70')),
 ('Rae Bareli', Decimal('141.69')),
 ('Binzhou', Decimal('141.67')),
 ('Atlixco', Decimal('141.67')),
 ('Tokat', Decimal('141.66')),
 ('Ikerre', Decimal('141.63')),
 ('Robamba', Decimal('140.69')),
 ('Qinhuangdao', Decimal('140.68')),
 ('Shimonoseki', Decimal('140.64')),
 ('Cuauhtmoc', Decimal('139.71')),
 ('Sterling Heights', Decimal('139.71')),
 ('Omiya', Decimal('139.70')),
 ('Yinchuan', Decimal('139.70')),
 ('Duisburg', Decimal('139.67')),
 ('Fengshan', Decimal('139.66')),
 ('Coatzacoalcos', Decimal('138.71')),
 ('Imus', Decimal('138.71')),
 ('Sungai Petani', Decimal('138.68')),
 ('Fontana', Decimal('138.67')),
 ('Siliguri (Shiliguri)', Decimal('138.66')),
 ('Ife', Decimal('137.70')),
 ('Parbhani', Decimal('137.69')),
 ('Aden', Decimal('137.69')),
 ('Ashdod', Decimal('137.67')),
 ('Nuuk', Decimal('137.66')),
 ('Ashgabat', Decimal('136.73')),
 ('Tychy', Decimal('136.64')),
 ('Athenai', Decimal('135.74')),
 ('Otsu', Decimal('135.74')),
 ('Wroclaw', Decimal('135.72')),
 ('Erlangen', Decimal('135.71')),
 ('Papeete', Decimal('135.70')),
 ('Sumqayit', Decimal('135.70')),
 ('Tabriz', Decimal('135.69')),
 ('Boksburg', Decimal('135.68')),
 ('NDjamna', Decimal('135.68')),
 ('Jinchang', Decimal('135.66')),
 ('Juiz de Fora', Decimal('134.73')),
 ('Angra dos Reis', Decimal('134.70')),
 ('Bhopal', Decimal('134.68')),
 ('La Plata', Decimal('134.68')),
 ('Leshan', Decimal('134.68')),
 ('Nha Trang', Decimal('134.68')),
 ('Kitwe', Decimal('134.67')),
 ('Lhokse

<div class="p-Widget jp-RenderedHTMLCommon jp-RenderedHTML jp-mod-trusted jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/html"><table>
    <tbody><tr>
        <th>city</th>
        <th>revenue</th>
    </tr>
    <tr>
        <td>Cape Coral</td>
        <td>221.55</td>
    </tr>
    <tr>
        <td>Saint-Denis</td>
        <td>216.54</td>
    </tr>
    <tr>
        <td>Aurora</td>
        <td>198.50</td>
    </tr>
    <tr>
        <td>Molodetno</td>
        <td>195.58</td>
    </tr>
    <tr>
        <td>Apeldoorn</td>
        <td>194.61</td>
    </tr>
    <tr>
        <td>Santa Brbara dOeste</td>
        <td>194.61</td>
    </tr>
    <tr>
        <td>Qomsheh</td>
        <td>186.62</td>
    </tr>
    <tr>
        <td>London</td>
        <td>180.52</td>
    </tr>
    <tr>
        <td>Ourense (Orense)</td>
        <td>177.60</td>
    </tr>
    <tr>
        <td>Bijapur</td>
        <td>175.61</td>
    </tr>
</tbody></table></div>

## 3.3 Insight 3 : Revenue of a movie by customer city and by month 

### 3.3.1 Total revenue by month

In [12]:
%%sql
SELECT sum(p.amount) as revenue, EXTRACT(month FROM p.payment_date) as month
from payment p
group by month
order by revenue desc
limit 10;

 * postgresql://student:***@127.0.0.1:5432/pagila
5 rows affected.


[(Decimal('28559.46'), 4.0),
 (Decimal('23886.56'), 3.0),
 (Decimal('9631.88'), 2.0),
 (Decimal('4824.43'), 1.0),
 (Decimal('514.18'), 5.0)]

### 3.3.2 Each movie by customer city and by month (data cube)

In [13]:
%%sql
SELECT f.title, p.amount, p.customer_id, ci.city, p.payment_date,EXTRACT(month FROM p.payment_date) as month
FROM payment p
JOIN rental r    ON ( p.rental_id = r.rental_id )
JOIN inventory i ON ( r.inventory_id = i.inventory_id )
JOIN film f ON ( i.film_id = f.film_id)
JOIN customer c  ON ( p.customer_id = c.customer_id )
JOIN address a ON ( c.address_id = a.address_id )
JOIN city ci ON ( a.city_id = ci.city_id )
order by p.payment_date
limit 10;

 * postgresql://student:***@127.0.0.1:5432/pagila
10 rows affected.


[('BLANKET BEVERLY', Decimal('2.99'), 130, 'guas Lindas de Gois', datetime.datetime(2017, 1, 24, 21, 21, 56, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0),
 ('FREAKY POCUS', Decimal('2.99'), 459, 'Qomsheh', datetime.datetime(2017, 1, 24, 21, 22, 59, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0),
 ('GRADUATE LORD', Decimal('3.99'), 408, 'Jaffna', datetime.datetime(2017, 1, 24, 21, 32, 5, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0),
 ('LOVE SUICIDES', Decimal('4.99'), 333, 'Baku', datetime.datetime(2017, 1, 24, 21, 33, 7, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0),
 ('IDOLS SNATCHERS', Decimal('6.99'), 222, 'Jaroslavl', datetime.datetime(2017, 1, 24, 21, 33, 47, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0),
 ('MYSTIC TRUMAN', Decimal('0.99'), 549, 'Santiago de Compostela', datetime.datetime(2017, 1, 24, 21, 36, 33, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0),
 ('SWARM GOLD', Decimal('1.99'), 269, 'Salinas', datetime.datetime(2017, 1, 24, 21, 40, 19, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0),
 ('LAWLESS VISION', Decimal('4.99'), 239, 'Ciomas', datetime.datetime(2017, 1, 24, 22, 0, 12, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0),
 ('MATRIX SNOWMAN', Decimal('4.99'), 126, 'Po', datetime.datetime(2017, 1, 24, 22, 29, 6, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0),
 ('HANGING DEEP', Decimal('5.99'), 399, 'Okara', datetime.datetime(2017, 1, 24, 22, 30, 47, 996577, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None)), 1.0)]

### 3.3.3 Sum of revenue of each movie by customer city and by month

TODO: Write a query that returns the total amount of revenue for each movie by customer city and by month. Limit the results to the top 10 movies. Your result should match the table below.

In [17]:
%%sql
SELECT f.title, ci.city, EXTRACT(month FROM p.payment_date) as month, sum(p.amount) as revenue
FROM payment p
JOIN rental r    ON ( p.rental_id = r.rental_id )
JOIN inventory i ON ( r.inventory_id = i.inventory_id )
JOIN film f ON ( i.film_id = f.film_id)
JOIN customer c  ON ( p.customer_id = c.customer_id )
JOIN address a ON ( c.address_id = a.address_id )
JOIN city ci ON ( a.city_id = ci.city_id )
group by f.title, ci.city, EXTRACT(month FROM p.payment_date)
order by sum(p.amount) desc
limit 10

 * postgresql://student:***@127.0.0.1:5432/pagila
10 rows affected.


[('CARIBBEAN LIBERTY', 'Ibirit', 4.0, Decimal('16.98')),
 ('EAGLES PANKY', 'Datong', 3.0, Decimal('16.98')),
 ('LUST LOCK', 'Coatzacoalcos', 4.0, Decimal('15.98')),
 ('INNOCENT USUAL', 'Valparai', 2.0, Decimal('13.98')),
 ('BREAKFAST GOLDFINGER', 'Ife', 4.0, Decimal('13.98')),
 ('VIRGIN DAISY', 'Cam Ranh', 4.0, Decimal('12.98')),
 ('TITANIC BOONDOCK', 'Sumqayit', 4.0, Decimal('12.98')),
 ('SLEEPLESS MONSOON', 'Fuyu', 4.0, Decimal('12.98')),
 ('TIES HUNGER', 'Naju', 3.0, Decimal('11.99')),
 ('SHOW LORD', 'Mannheim', 1.0, Decimal('11.99'))]

<div class="p-Widget jp-RenderedHTMLCommon jp-RenderedHTML jp-mod-trusted jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/html"><table>
    <tbody><tr>
        <th>title</th>
        <th>city</th>
        <th>month</th>
        <th>revenue</th>
    </tr>
    <tr>
        <td>SHOW LORD</td>
        <td>Mannheim</td>
        <td>1.0</td>
        <td>11.99</td>
    </tr>
    <tr>
        <td>AMERICAN CIRCUS</td>
        <td>Callao</td>
        <td>1.0</td>
        <td>10.99</td>
    </tr>
    <tr>
        <td>CASUALTIES ENCINO</td>
        <td>Warren</td>
        <td>1.0</td>
        <td>10.99</td>
    </tr>
    <tr>
        <td>TELEGRAPH VOYAGE</td>
        <td>Naala-Porto</td>
        <td>1.0</td>
        <td>10.99</td>
    </tr>
    <tr>
        <td>KISSING DOLLS</td>
        <td>Toulon</td>
        <td>1.0</td>
        <td>10.99</td>
    </tr>
    <tr>
        <td>MILLION ACE</td>
        <td>Bergamo</td>
        <td>1.0</td>
        <td>9.99</td>
    </tr>
    <tr>
        <td>TITANS JERK</td>
        <td>Kimberley</td>
        <td>1.0</td>
        <td>9.99</td>
    </tr>
    <tr>
        <td>DARKO DORADO</td>
        <td>Bhilwara</td>
        <td>1.0</td>
        <td>9.99</td>
    </tr>
    <tr>
        <td>SUNRISE LEAGUE</td>
        <td>Nagareyama</td>
        <td>1.0</td>
        <td>9.99</td>
    </tr>
    <tr>
        <td>MILLION ACE</td>
        <td>Gaziantep</td>
        <td>1.0</td>
        <td>9.99</td>
    </tr>
</tbody></table></div>